packages/mdc-chips/_mixins.scss

```diff
@@ -429,6 +448,14 @@ $mdc-chip-ripple-target: ".mdc-chip__ripple";
     padding-left: $horizontal-padding-value;
     border-width: $width;
   }
+
+  #{$mdc-chip-ripple-target} {
+    @include mdc-feature-targets($feat-structure) {
+      top: -$width;
```

Comment:
top & left can be set to `0` which is the same thing.

Reply:
I think this is necessary in order to compensate for the chip border. top/left of 0 will position the ripple element inside the border.

Reply:
Ah that's right. this is to offset transparent border-width added to ripple.